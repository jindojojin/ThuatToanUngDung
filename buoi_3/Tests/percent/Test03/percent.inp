50
42 8 35 1 14
20 5 29 9 37
13 5 6 6 70
32 8 12 2 46
46 3 28 7 16
42 5 3 4 46
43 3 22 7 25
19 6 48 7 20
22 9 20 3 46
18 10 36 5 31
4 2 23 4 67
24 5 42 2 27
4 9 48 5 34
13 8 38 10 31
24 2 30 9 35
17 6 41 3 33
39 7 41 3 10
15 9 47 6 23
41 10 21 1 27
7 2 44 9 38
30 4 35 5 26
7 1 17 7 68
32 9 45 10 4
27 4 38 9 22
19 3 30 2 46
34 6 40 9 11
5 1 28 7 59
24 7 22 6 41
25 3 21 10 41
28 4 48 3 17
37 1 12 7 43
6 8 6 5 75
32 3 1 1 63
42 5 17 1 35
8 2 8 8 74
8 8 4 4 76
46 10 10 9 25
22 9 23 7 39
7 1 14 9 69
1 2 13 6 78
11 10 25 8 46
50 0 50 0 0
3 1 42 7 47
25 1 47 2 25
49 10 19 5 17
32 5 4 10 49
19 9 1 9 62
28 8 29 4 31
49 4 8 2 37
11 8 14 5 62
