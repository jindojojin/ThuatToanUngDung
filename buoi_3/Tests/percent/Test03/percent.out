100
100
100
50
100
100
100
100
100
100
100
100
100
100
100
100
100
100
100
100
100
100
100
100
100
100
100
100
100
100
100
100
100
100
50
25
100
100
100
100
100
2
100
100
100
100
100
100
100
100
